lib/crowbar/client/request/base.rb

```diff
@@ -50,7 +50,7 @@ def params
           when :post
             [
               method,
-              content.to_json
+              content
```

Comment:
> but as usual there are exceptions like the upload

Isn't backups/upload the only exception here? I.e. wouldn't it be better to have backup/upload.rb override the ```params``` method?